templates/android-dot-com/styles.css

```diff
@@ -481,7 +481,8 @@ a img{
     transition: opacity 0.2s cubic-bezier(0.4, 0, 0.2, 1);
   }
 
-  .android-search-box.is-focused ~ .android-mobile-title {
+  .android-search-box.is-focused + .android-navigation-container +
+      .android-mobile-title {
```

Comment:
Thanks for the link, @Garbee! Both the android.com template and your test case look correct. Here's the latter:

![image](https://cloud.githubusercontent.com/assets/409615/10426285/7e2ea390-70d6-11e5-8399-f8569ff8e551.png)